### PR TITLE
Major Repository Rennovation, Branch: develop

### PR DESCRIPTION
**Pull request by TrainDoctor**
This revamps the develop branch and integrates a bunch of changes I am proposing to bring to this repo.

This branch brings Github issue templates and merges commits to get everything mostly parallel.

Preview the changes [here](https://github.com/TrainDoctor/RL-Expanded-Rewrite)

### CODE REVIEW

File: game/_script_/actors/Actor.rpy [hunk 1]
```diff
@@ -38,25 +38,15 @@
...
+    (this is set by Renpy devs, not us)
     DIALOG (say)
         (who & what, prefixes and suffixes)
         (html tags)
         (transitions)
         (history)
-    CHARACTER IMAGE (images)
+    CHARACTER IMAGE (images) (Custom LayeredImage)
     PREDICTION (images)
         (image pre-caching)
```
Comment (pervyrogue) [inline game/_script_/actors/Actor.rpy]: This needs moved out of renpy files and into an issue.

Documentation of files should follow docblock formatting.

**Review by pervyrogue — COMMENTED** [review]
Move of major mechanics to separate files is great.

**Review by pervyrogue — CHANGES_REQUESTED** [review]
Why did we remove this?

Oni specific items can be removed when repo is forked and sanitized for use in other projects.

**Review by pervyrogue — COMMENTED** [review]
Image Test worked. More documentation of test setup would have been preferred.

Note to self: Comment more inside renpy specific scripting.

File: game/Locations.rpy [hunk 2]
```diff
@@ -76,15 +76,15 @@ init -2 python:
 
 
 # Test definition using Location.
-define university_square = Location("University Square", adjacent=
+define testenv = Location("University Square", adjacent=
```
Comment (pervyrogue) [inline game/Locations.rpy]: Why did we remove university_square? testenv should have been a separate define.

File: game/Events.rpy [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+image study_day = "StudyDay.jpg"
+
+define p = Character("Zero",
+                     who_color="#000000")
+
+$ PlayerSkinTone = "#028800"
+
+label intro:
+    $ renpy.scene()
+    $ renpy.show("study_day")
```
Comment (pervyrogue) [inline game/Events.rpy]: Avoid equivalent statements in renpy scripts unless using functions/variables/etc.

File: game/screens.rpy [hunk 4]
```diff
@@ -300,6 +300,7 @@ screen main_menu():
 
         textbutton _("Start Game") action Start()
         textbutton _("Load Game") action ShowMenu("load")
+        textbutton _("Test Enviroment") action Start("testenv")
```
Comment (pervyrogue) [inline game/screens.rpy:303]: Good use of the main menu. Add a conditional for display reliant on config.developer

File: game/Locations.rpy [hunk 5]
```diff
@@ -74,11 +74,47 @@ init -2 python:
...
+        
+        "Chat":
+            pass
+        "Study":
+            pass
+        "Lock Door": # TrainDoc: Originally called "Lock the door"
+            pass
+        "Sleep":
+            pass
+        "Shop": # TrainDoc: wtf are we going to do w/ the shop? it's so half baked lol
```
Comment (pervyrogue) [inline game/Locations.rpy:103]: Create related issue and use REF or SEE codetag with issue number.

File: game/Events.rpy [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+define r = Character("Rogue",
+                     who_color="#85bb65")
+
+$ PlayerSkinTone = "#028800"
+
+label intro:
+    $ renpy.scene()
+    $ renpy.show("study_day")
+
+    # TrainDoc:
```
Comment (pervyrogue) [inline game/Events.rpy]: Use author docblock param instead of name tags in comments.

**Review by pervyrogue — CHANGES_REQUESTED** [review]
Discuss and change file structure to meet #80.

**Comment by TrainDoctor** [comment]
> Why did we remove this?
> 
> Oni specific items can be removed when repo is forked and sanitized for use in other projects.

This doesn't have any code attached to it so I'm not sure what needs to be changed.

**Comment by TrainDoctor** [comment]
I've resolved to remove all the non-github issue related changes and from my fork and to attempt another pull request.

**Pull request closed without merging**